RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -704,6 +704,8 @@ void preAdvUpdateFamiliar(location place)
 		handleFamiliar(get_property("auto_100familiar").to_familiar());			//do not break 100 familiar runs
 	}
 	
+	auto_forceEagle(); // force Patriotic Eagle if we have a >0 combats until we can screech again
```

Comment:
I moved it to basically the top of that function, though that might be too high. I do think it should be fairly high priority though because of how useful it's banish is